components/esp_websocket_client/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: "0.0.1"
+description: esp modem
+dependencies:
+  # Required IDF version
+  idf:
+    version: ">=4.1"
```

Comment:
Not sure if this will work with `v4.1`, IDF < v5.0 have the same ws client already in the default component paths.